src/features/authentication/logout/ui/LogoutButton/LogoutButton.tsx

```diff
@@ -0,0 +1,19 @@
+import {useAppDispatch} from '@/shared/model'
+import {logoutThunk,} from '../../model/logout'
+import {Link} from "react-router-dom";
+import {RefreshToken} from "@/shared/model/types.ts";
+
+
+export function LogoutButton(token: RefreshToken) {
+  const dispatch = useAppDispatch()
+
+  const onConfirmLogout = async (e: React.MouseEvent<HTMLElement>) => {
+    e.stopPropagation()
+    e.preventDefault()
+    dispatch(logoutThunk(token)).unwrap()
+  }
+
+  return (
+      <Link to="/" onClick={onConfirmLogout}>Выйти</Link>
```

Comment:
Why not use a button here?

Reply:
I don't know what is better, here it's just a simple redirect basically